.github/workflows/autorelease-rubygem.yml

```diff
@@ -61,8 +61,13 @@ jobs:
           exit 0
         fi
 
+        if [ -z "$(git log "v$curr_ver..main" --no-merges --format=format:%an | sort | uniq)" ]; then
```

Comment:
Given that you're just checking for zero-length output, the `sort` and `uniq` are redundant and a little confusing.

Reply:
Ahh yeah that was a bad copy paste situation, have removed it